index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt('Ingrese su nombre ');
+alert("Hola ${nombreUsuario}!");
```

Comment:
En este caso seria mejor no usar interpolacion de strings
```suggestion
alert("Hola " + nombreUsuario);
```